dev/io.openliberty.mcp.internal/src/io/openliberty/mcp/internal/meta/package-info.java

```diff
@@ -0,0 +1,13 @@
+/*******************************************************************************
+ * Copyright (c) 2025 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+/**
+ *
+ */
+package io.openliberty.mcp.internal.meta;
```

Comment:
We need the `@TraceOptions` annotation that all the other `package-info.java` files have here.